### PR TITLE
Readme part2

### PR DESCRIPTION
**Pull request by Spill-Tea**
Finalize edits to Readme Doc

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,55 +1,57 @@
...
-regarding current implementation details to describe design decisions and discuss
-various development issues and solutions on our readthedocs page,
-found [here](https://fhy.readthedocs.io/en/latest/design_blog/index.html). We encourage
-new contributors and users to review the blog for more information regarding *FhY* and
-get involved.
+### *FhY* Design Blog
+The *FhY developers* provide a transparent "***Glass-House***" design blog to openly
+describe current implementation details and software design decisions. We also discuss
+various issues encountered during development and solutions on our [*Read the Docs* page](https://fhy.readthedocs.io/en/latest/design_blog/index.html). We encourage new contributors and
+users to review the design blog for more information regarding *FhY* and as a starting
```
Comment (christopherpriebe) [inline README.md:36]: end of sentence does not make sense; seems like it was split from prior sentence

**Review by christopherpriebe — CHANGES_REQUESTED** [review]
change one thing

File: README.md (hunk 1 above)
Comment (Spill-Tea) [reply README.md]: I have updated paragraph to be less of a run-on sentence at the end.

**Review by christopherpriebe — APPROVED** [review]
LGTM

**Pull request merged**